core/src/main/resources/reference.conf

```diff
@@ -33,13 +33,13 @@ gearpump {
     enabled = false
 
     # We will take one metric out of ${sample.rate}
-    sample-rate = 10
+    sample-rate = 1
 
     report-interval-ms = 15000
 
     # reporter = "graphite"
     # reporter = "akka"
-    reporter = "logfile"
+    reporter = "akka"
```

Comment:
Our default reporter is set to "akka" in gear.conf. This is to make sure reference.conf is consistent with gear.conf.

Setting reporter as "akka" means we will gather all metrics by sending it to a actor receiver.